client/src/pages/certification.css

```diff
@@ -216,3 +216,20 @@
     margin-top: 20px;
   }
 }
+
+.certificate-outer-wrapper {
+  margin-top: calc(-1 * var(--header-height));
+  background-color: #fff;
+}
+
+.certificate-outer-wrapper .row,
+.certificate-outer-wrapper .information,
+.certificate-outer-wrapper .verify {
+  background-color: #fff;
+}
+
+.certificate-outer-wrapper .donation-completion-buttons button {
```

Comment:
Probably a stupid question, but what's the best way to test this?  I don't really want to spam fake donations just to see the buttons.

Reply:
I just manually wrote the CSS in the <head> after using a test card number. Not sure if there’s another way other than getting to to run locally.

Reply:
Nevermind, I decided one obviously fake donation would be okay.  This looks good 👍